### DISCUSSION TITLE
Idea of adding Difficulty levels

### DISCUSSION THREAD

**Discussion by ageekhere** [OpenRCT2/OpenRCT2#17893] [Feature requests]
I wonder if adding difficulty levels would be a cool idea to make the missions more challenging. Idea of changes to the difficulty  could be
1. Guess stats drop quicker (happiness, energy)
2. Guess enter the park with less money
3. Interest rates for loans are higher
4. Guess become more picky on going on the same ride again
etc

Maybe have 6 levels
1 Stranded 
2 Hard
3 Harder
4..
5..
6..Hardest (at this level it must be extremally hard to win)
When you complete a park it lists what level you played at and the high score for that level.

**Comment by James103** [comment]
This would be better suited for a plugin instead of being integrated into the base game, IMO.

**Reply by jjpb44** [reply]
Something like what RCT3 have? Bronce, Silver and Gold Medals for challenging objectives in each scenario.

**Reply by ageekhere** [reply]
Yes, this would work quite well
